### ISSUE TITLE
Typing DebugView.debugView is now needed...

### ISSUE THREAD

**Issue by marklnichols** [ndmitchell/debug#49]
Can we reconfigure things so that only 'debugView' needs to be entered as per the current documentation?

**Comment by ndmitchell** [comment]
As far as I can tell, the current docs still work. What problem are you seeing?

**Comment by marklnichols** [comment]
I get:

interactive :1:1: error:
    * Variable not in scope: debugView
    * Perhaps you meant `Debug.debugView' (imported from Debug)

**Comment by ndmitchell** [comment]
What instructions are you following? Any deviations?

**Comment by marklnichols** [comment]
It seems that all such problems go away if I specifically load the Debug module via 
:m + Debug

The different form of the error message (with the "Perhaps you meant...") made me think it was a different problem...so it is covered by the documentation (via the comment in the FAQ)

I seem to always need to specifically load Debug via. :m -- do you not need to do that?  I can't find anything I'm doing differently from the instructions, other than using stack commands for ghci, etc.

Maybe that ":m + Debug" should be part of the main instructions rather than part of the FAQ?

**Comment by ndmitchell** [comment]
Which instructions are you following? I'm following the ones at https://github.com/ndmitchell/debug#haskell-debugger----

**Comment by marklnichols** [comment]
Yes, those same instructions.  Could it be a difference between calling ghci directly vs stack ghci?

**Comment by ndmitchell** [comment]
Which `debug` is `stack ghci` picking up? Does the issue go away if you run `ghci` directly? What happens if you run `stack exec ghc-pkg list`?
